users/hvp/tap_dances.c

```diff
@@ -0,0 +1,75 @@
+#include "tap_dances.h"
+
+// Tap dance function for enable swedish characters on first layer. Unregister to not let tap bleed over to next keypress.
+// Tap dance 1
+void dance_1_finished(qk_tap_dance_state_t *state, void *user_data) {
+    if (state->count == 2) {
+        tap_code(KC_SCLN);
+    } else {
+        register_code(KC_RALT);
+        register_code(KC_O);
+        unregister_code(KC_RALT);
+        unregister_code(KC_O);
```

Comment:
These can be reduced to:

```c
tap_code16(ALGR(KC_O));
```

Reply:
I have tested and no, unfortunately, does it not work.  The end result is not the same when change over to use tap_code16(ALGR(KC_O));

Reply:
How so? It should be doing the same thing, though the RALT is now added to the modifier section of the keyboard report, instead of as a keycode, which IMO is better.

Reply:
Im fully aware that it should do the same thing but the end result is that it dont work using the tap_code16 combo.